docs/architecture/languages.md

```diff
@@ -0,0 +1,32 @@
+(languages)=
+(language-hosts)=
+# Language hosts
+
+*Language hosts*, or *language runtimes*, are the means by which Pulumi is able
+to support a variety of programming languages. Officially, the term "language
+host" refers to the combination of two parts:
```

Comment:
> Officially, the term "language host" refers to the combination of two parts:

Does it? I thought host was always just the plugin and that "runtime" was the overloaded one that could mean just the sdk library, or the combination of both to allow things to work.

Reply:
I was under the same impression, that language hosts are just the means to run the language plugin.

Reply:
I agree, but our public stance is currently different, so I've stayed consistent for now https://www.pulumi.com/docs/concepts/how-pulumi-works/#language-hosts

Reply:
Good to stay consistent in that case.